assets/Zed/js/modules/form-toggler.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+'use strict';
+
+$(document).ready(function() {
+    $('.dependent-trigger').each(function (index, selectItem) {
```

Comment:
Suggest to use better naming instead of `dependent-trigger`, `dependent-child`, `dependent-parent`, etc.

Reply:
Fixed.